code/modules/surgery/bodyparts/species_parts/misc_bodyparts.dm

```diff
@@ -88,6 +88,35 @@
 /obj/item/bodypart/r_leg/slime
 	limb_id = SPECIES_SLIMEPERSON
 
+//SKYRAT ADDITION BEGIN -  Roundstartslime bodyparts for Kapulimbs
+///ROUNDSTARTSLIME
+/obj/item/bodypart/head/roundstartslime
+	limb_id = SPECIES_SLIMEPERSON //same icon state, no real reason to make a new define
+	is_dimorphic = TRUE
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/slime_parts_greyscale.dmi' //Sprite override to not use default slime sprites
+
+/obj/item/bodypart/chest/roundstartslime
+	limb_id = SPECIES_SLIMEPERSON
+	is_dimorphic = TRUE
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/slime_parts_greyscale.dmi'
+
+/obj/item/bodypart/l_arm/roundstartslime
+	limb_id = SPECIES_SLIMEPERSON
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/slime_parts_greyscale.dmi'
+
+/obj/item/bodypart/r_arm/roundstartslime
+	limb_id = SPECIES_SLIMEPERSON
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/slime_parts_greyscale.dmi'
+
+/obj/item/bodypart/l_leg/roundstartslime
+	limb_id = SPECIES_SLIMEPERSON
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/slime_parts_greyscale.dmi'
+
+/obj/item/bodypart/r_leg/roundstartslime
+	limb_id = SPECIES_SLIMEPERSON
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/slime_parts_greyscale.dmi'
+
+//SKYRAT ADDITION END
```

Comment:
You can move all of that to a modular file instead, it'll work just the same! (just make sure they're ticked and we're good)